### ISSUE TITLE
Weird tag for map

### ISSUE THREAD

**Issue by Alicia-bites** [Alicia-bites/cub3d#3]
This map is called : ok_all_mini_size.cub

```
NO ./texture/no.xpm
SO ./texture/so.xpm
WE ./texture/we.xpm
EA ./texture/ea.xpm
F 220,100,0
C 225,30,0
111111
111111
111111
```

I think it should be a ko...?

**Comment by BarrierAntho** [comment]
Ohhh shit, i forgot to set the player spawn.
I will update

**Comment by BarrierAntho** [comment]
fixed with da40b8a48caa7b83af6ae580ebb6ad157c446931
